src/pages/Portfolio/LendingStats/LendingStatsPage.vue

```diff
@@ -178,6 +178,11 @@ export default {
 
 			return completedBadgesArr;
 		},
+		totalPossibleBadges() {
```

Comment:
This is no longer being used.

Reply:
Actually it is still being used by the other component. Is there an issue passing the larger number? How is it used in the other component?

Reply:
I think loading will always be true now.

Reply:
If we aren't showing that value, shouldn't we just remove it entirely and use `completedAchievements` as the loading indicator?

Reply:
If we're using a field no longer shown anywhere (or used for a displayed value) as the loading indicator, we need to find a more accurate indicator.